code/modules/mob/living/silicon/ai/decentralized/management/ai_controlpanel.dm

```diff
@@ -320,7 +320,7 @@ GLOBAL_VAR_INIT(ai_control_code, random_nukecode(6))
 				one_time_password_used = TRUE
 				var/msg = "<h4>Warning!</h4><br>We have detected usage of the AI Control Code for unlocking a console at coordinates ([src.x], [src.y], [src.z]) by [usr.name]. Please verify that this is correct. Be aware we have cancelled the current control code.<br>\
 				If needed a new code can be printed at a communications console."
-				priority_announce(msg, sender_override = "Central Cyber Security Update", has_important_message = TRUE, sanitize = FALSE)
+				priority_announce(msg, sender_override = "Central Cyber Security Update", has_important_message = TRUE)
```

Comment:
Does this still work? sanitize might be off for a reason.

Reply:
will double check but it worked when I tested it